File: programmers/level1/015.py
# 15. 서울에서 김서방 찾기
def solution(seoul):
    answer = ''
    where = 0 
    for i in range(len(seoul)):
        if seoul[i] == 'Kim':
            where=i
    answer = "김서방은 "+ str(where)+ "에 있다"
    return answer

File: programmers/level0/015.java
//15. 배열 뒤집기
class Solution {
    public int[] solution(int[] num_list) {
        int n = num_list.length;
        int[] answer = new int[n];
        
        for(int i=0; i<n; i++){
            answer[n-i-1]=num_list[i];
        }
        
        return answer;
    }
}